cranelift/codegen/src/timing.rs

```diff
@@ -273,14 +278,18 @@ mod enabled {
 #[cfg(not(feature = "timing"))]
 mod disabled {
     use super::{DefaultProfiler, Pass, Profiler};
-    use std::any::Any;
-    use std::boxed::Box;
+    use alloc::boxed::Box;
+    use core::any::Any;
 
     impl Profiler for DefaultProfiler {
         fn start_pass(&self, _pass: Pass) -> Box<dyn Any> {
             Box::new(())
         }
     }
+
+    pub fn start_pass(_pass: Pass) -> Box<dyn Any> {
+        Box::new(())
+    }
```

Comment:
1. where is this function used?
2. It should probably be `pub(crate)` instead.

Reply:
yup, good catch 👍🏻